runtime/gc_base/GCExtensions.cpp

```diff
@@ -235,46 +235,46 @@ MM_GCExtensions::identityHashDataRemoveRange(MM_EnvironmentBase *env, MM_MemoryS
 }
 
 void
-MM_GCExtensions::updateIdentityHashDataForSaltIndex(UDATA index)
+MM_GCExtensions::updateIdentityHashDataForSaltIndex(uintptr_t index)
 {
 	getJavaVM()->identityHashData->hashSaltTable[index] = (U_32)convertValueToHash(getJavaVM(), getJavaVM()->identityHashData->hashSaltTable[index]);
 }
 
 /*
- * computeDefaultMaxHeapForJava is for Java only, it will be called during gcParseCommandLineAndInitializeWithValues(),
- *  computeDefaultMaxHeap() will still be called during MM_GCExtensionsBase::initialize(), computeDefaultMaxHeapForJava() can overwrite value of memoryMax.
+ * ComputeDefaultMaxHeapForJava is for Java only, it will be called during gcParseCommandLineAndInitializeWithValues(),
+ * ComputeDefaultMaxHeap() will still be called during MM_GCExtensionsBase::initialize(), computeDefaultMaxHeapForJava() can overwrite value of memoryMax.
```

Comment:
Why this change? Method names are not capitalized

Reply:
Oh, first I want to remove the second line space, and I'm not aware that the method name should not be capitalized. Will revert this change.